test/jsonpointer/jsonpointer_weakpointer_test.cc

```diff
@@ -0,0 +1,272 @@
+#include <gtest/gtest.h>
+#include <sourcemeta/jsontoolkit/jsonpointer.h>
+#include <type_traits>
+
+const std::string foo = "foo";
```

Comment:
```suggestion
static const std::string foo = "foo";
```

Can you make these static so that they are not visible from other files from the same compilation unit?